packages/shared-ui/src/styles/index.css

```diff
@@ -1196,6 +1196,63 @@ button.cb-phone-otp-resend-button {
   height: 0.9rem;
 }
 
+.cb-last-identifier {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+  border-radius: var(--cb-border-radius-sm);
+  border: var(--cb-border-color) 1px solid;
+  margin: 0.75rem 0;
+  padding: 0.6rem 0.5rem 0.6rem 1rem;
+  align-self: center;
+  cursor: pointer;
+  width: 83%;
```

Comment:
I agree.. Will create a ticket for this. One thing is that it is not always possible to achieve proper responsive sizes without using %age in case of block widths. But I agree we might not need soo many places. I will see how we can improve this.